packages/task/src/browser/task-service.ts

```diff
@@ -947,7 +975,13 @@ export class TaskService implements TaskConfigurationClient {
             }
         );
         this.shell.addWidget(widget, { area: 'bottom' });
-        this.shell.activateWidget(widget.id);
+        if (taskInfo && taskInfo.config.presentation && taskInfo.config.presentation.reveal === RevealKind.Always) {
+            if (taskInfo.config.presentation.focus) { // assign focus to the terminal if presentation.focus is true
+                this.shell.activateWidget(widget.id);
+            } else { // show the terminal but not assign focus
```

Comment:
`else if (taskInfo.config.presentation.reveal === RevealKind.Always)` and remove `taskInfo.config.presentation.reveal === RevealKind.Always` from outer if?

Reply:
if `(reveal === 'never' || reveal === 'silient') && focus === true`, the terminal does not get the focus. That's why I am putting `taskInfo.config.presentation.reveal === RevealKind.Always` in the outter if.

![Peek 2020-01-06 23-48](https://user-images.githubusercontent.com/37082801/71869172-0d778b80-30df-11ea-82a4-3403c9bf50b8.gif)

Reply:
👍